<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KoFileDialogTester</class>
 <widget class="QWidget" name="KoFileDialogTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>396</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QLabel" name="lblMime">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="bnOpenFile">
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="bnOpenFiles">
     <property name="text">
      <string>Open Files</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unique key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtUniqueKey">
       <property name="text">
        <string>blablabla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Filter:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Results:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listResults"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioMime">
        <property name="text">
         <string>Mime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioName">
        <property name="text">
         <string>Named</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkHideNameFilterDetailsOption">
        <property name="text">
         <string>Hide name filter details option</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkSetDefaultFilter">
        <property name="text">
         <string>Set default filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="bnSaveFile">
     <property name="text">
      <string>Save File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="bnImportFiles">
     <property name="text">
      <string>Import Files</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="bnImportFile">
     <property name="text">
      <string>Import File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="bnOpenDirectory">
     <property name="text">
      <string>Open Directory</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="bnImportDirectory">
     <property name="text">
      <string>Import Directory</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>bnImportFile</zorder>
  <zorder>bnOpenFile</zorder>
  <zorder>bnOpenFiles</zorder>
  <zorder>bnOpenDirectory</zorder>
  <zorder>bnImportFiles</zorder>
  <zorder>bnImportDirectory</zorder>
  <zorder>bnSaveFile</zorder>
  <zorder>groupBox</zorder>
  <zorder>lblMime</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
